README.md

```diff
@@ -82,7 +82,7 @@ Visit [our quick start guide](https://earthaccess.readthedocs.io/en/latest/quick
 
 ## Compatibility
 
-Only **Python 3.8+** is supported.
+Only **Python 3.9+** is supported.
```

Comment:
I think the `+` is easy to miss, so it is easy to read the original wording as though _exactly_ Python 3.9 is supported (and nothing else), so this is perhaps a bit more clear:

```suggestion
The _minimum_ supported Python version is **3.9**.
```

Reply:
Agreed!